src/Session.fz

```diff
@@ -269,14 +269,65 @@ module Session (sessions lock_free.Map String Session,
     send_event event html_encoded_data
 
 
+  # NYI: BUG: this should be in base lib
+
+  # split sequence at s, if there is no limit, otherwise if limit is an integer n,
+  # for at most n occurrences of s
+  #
+  # if split_after is true, all but the last element of the resulting list include
+  # the separator
+  #
+  # helper feature which unifies the code of the different split features in one
+  #
```

Comment:
this is just the comment from `split0` in `Sequence/equatable.fz` in the base lib. what's the difference here? elaborate in the comment.

Reply:
Base lib returns a list of Strings, it does the splitting lazily. This does eager splitting but once only.

Reply:
I think that deserves a comment in the code too, not just in this PR.